lib/flight_job/models/script.rb

```diff
@@ -197,22 +233,33 @@ def render(**answers)
     end
 
     # XXX: Eventually the answers will likely be saved with the script
```

Comment:
This comment can be removed.

Reply:
c2968f93078c9098653107f48c025a1bf2112d29